MatomoTracker/MatomoTracker.swift

```diff
@@ -462,3 +462,15 @@ extension MatomoTracker {
         matomoUserDefaults.copy(from: UserDefaults.standard)
     }
 }
+
+extension MatomoTracker {
+    /// Track an outlink
+    ///
+    /// - Parameters:
+    ///   - url: The url of the outlink
+    @objc public func trackOutlink(url: URL) {
+        var customTrackingParameters = [String:String]()
+        customTrackingParameters["link"] = url.absoluteString
+        track(Event(tracker: self, action: [], url: url, customTrackingParameters: customTrackingParameters))
```

Comment:
The `customTrackingParameters` dictionary was meant for actual custom tracking parameters. Since in this case this isn't really a custom tracking parameter if it is defined in the SDK, what do you think about if we move this parameter into the Event itself?